noir/noir-repo/acvm-repo/acvm_js/test/node/build_info.test.ts

```diff
@@ -7,7 +7,7 @@ it('returns the correct build info', () => {
   let revision: string;
 
   try {
-    revision = child_process.execSync('git rev-parse HEAD').toString().trim();
+    revision = process.env.GIT_COMMIT || child_process.execSync('git rev-parse HEAD').toString().trim();
```

Comment:
makes a test pass with our fakey git commit

Reply:
(if we dont use a fake hash, noir wasm/native builds can differ due to cache and noir tests dont like that)